ur_robot_driver/config/ur10_controllers.yaml

```
@@ -130,6 +130,36 @@ joint_group_vel_controller:
    type: velocity_controllers/JointGroupVelocityController
    joints: *robot_joints
 
+forward_joint_traj_controller:
+    type: "joint_trajectory_controllers/PassThroughController"
```

```suggestion
    type: joint_trajectory_controllers/PassThroughController
```

quotes are not needed for simple strings.